Tests/Functional/Bundle/TestBundle/Controller/SerializerErrorController.php

```diff
@@ -32,6 +32,7 @@ public function exceptionAction()
 
     /**
      * @View
+     * @group legacy
```

Comment:
The problem is that this tests use a deprecated API, so if I don't flag this tests as legacy, there is an error.